grafana.version

```diff
@@ -1 +1 @@
-6.6.1
+6.7.1
```

Comment:
Meanwhile, it is 6.7.2, should we risk it :)

Reply:
6.7.2 is what I tested on, honestly, I'll bump it after I merge.